components/Tooltip/Tooltip.unit.test.jsx

```diff
@@ -2,25 +2,43 @@ import React from 'react';
 import { mount } from 'enzyme';
 import Tooltip from './Tooltip';
 
+const TOOLTIP_TEXT = 'This is a hint';
+
 describe('Tooltip component ', () => {
   describe('All positions', () => {
     it('Should match the snapshot when place is top', () => {
-      const tooltip = mount(<Tooltip place="top">Hover Me</Tooltip>);
+      const tooltip = mount(
+        <Tooltip place="top" text={TOOLTIP_TEXT}>
+          Hover Me
+        </Tooltip>,
+      );
       expect(tooltip.html()).toMatchSnapshot();
```

Comment:
I think you can use the `shallow` method from enzyme and remove this `.html()`.

Reply:
Thanks @allyssonsantos! Lets create a issue so we can review this tests